router.js

```diff
@@ -0,0 +1,43 @@
+
+function navigateTo(url) {
+    // Update the browser's history stack
+    history.pushState(null, null, url);
+    router();
+  }
+
+
+function router() {
+    // Get the current path
+    const path = window.location.pathname;
+
+    if (path === '/help') {
+        document.getElementById('help-page').style.display = 'block';
```

Comment:
Rather than hiding all the sections individually, can you swap out <main>. That way only one thing changes.

Reply:
@anderspitman what do you mean "swap out"?

Reply:
Please see the issue #102 and let me know if that what you want.